scripts/docs/templates/monitor-page.md.tmpl

```diff
@@ -41,40 +42,35 @@ This monitor has no configuration options.
 {{with .metrics -}}
 ## Metrics
 
-The following table lists the metrics available for this monitor. Metrics that are marked as Included are standard metrics and are monitored by default.
+The following table lists the metrics available for this monitor.
+{{ if not $sendAll -}}
+Metrics that are categorized as [container/host/bundled](https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-bundled-and-high-resolution-metrics)
+are marked as _Default_ in the table below.
+{{- else -}}
+This monitor emits all metrics by default; however, **none are categorized as [container/host/bundled](https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-bundled-and-high-resolution-metrics)**.
+{{end}}
 
-| Name | Type | Included | Description |
-| ---  | ---  | ---    | ---         |
+| Name | Type {{if not $sendAll}}| [Default](https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-bundled-and-high-resolution-metrics) {{end}}| Description |
+| ---  | ---  {{if not $sendAll}}| ---    {{end}}| ---         |
 {{ range $metric, $info := . -}}
-| `{{$metric}}`{{if has $info "alias"}}<br>_or_ `{{$info.alias}}`{{end}} | {{$info.type}} | {{if $info.included}}✔{{end}} | {{$info.description | strings.TrimSpace}} |
+| `{{$metric}}`{{if has $info "alias"}}<br>_or_ `{{$info.alias}}`{{end}} | {{$info.type}} {{if not $sendAll}}| {{if $info.included}}✔{{end}} {{end}}| {{$info.description | strings.TrimSpace}} |
 {{end}}
 {{end -}}
 
-{{- $noWhitelistMonitors := slice "elasticsearch" -}}
-
-{{if and .metrics (not .sendAll) (not (has $noWhitelistMonitors .monitorType))}}
-To specify custom metrics you want to monitor, add a `metricsToInclude` filter
-to the agent configuration, as shown in the code snippet below. The snippet
-lists all available custom metrics. You can copy and paste the snippet into
-your configuration file, then delete any custom metrics that you do not want
-sent.
-
-Note that some of the custom metrics require you to set a flag as well as add
-them to the list. Check the monitor configuration file to see if a flag is
-required for gathering additional metrics.
+{{ if $sendAll }}
+The agent does not do any built-in filtering of metrics coming out of this
+monitor.
+{{ end -}}
 
-```yaml
+{{ if and .metrics (not .sendAll) }}
 
-metricsToInclude:
-  - metricNames:
-{{- range $metric, $info := .metrics -}}
-{{if not $info.included}}
-    - {{ $metric }}
-{{- end -}}
-{{end}}
-    monitorType: {{.monitorType}}
-```
-{{end}}
+### Built in filtering
+This monitor will perform built-in filtering if you are using agent version
+4.7.0+ and have the `enableBuiltInFiltering: true` option set at the top-level
```

Comment:
This goes along with my comment about having separate topics for filtering and extra metrics. I think it's confusing here that to get more metrics you have to go to a page called Filtering. Filtering implies you want less of something when you're trying to actually get more of something.

It also means the option `enableBuiltInFiltering` is confusing if you're thinking in the context of configuring additional metrics. (Even though in implementation details it makes perfect sense to us).

Reply:
Ok, I moved all of the relevant parts about extra/custom metrics to the monitor docs and left the "filtering" doc to just contain stuff about excluding datapoints.  That does seem to be clearer.